src/cswinrt/Directory.Build.targets

```diff
@@ -1,5 +1,17 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+    <PropertyGroup>
+        <MidlRTCopyWinMDToOutputDirectoryDependsOn>$(MidlRTCopyWinMDToOutputDirectoryDependsOn);CopyWinRTInteropWinMD</MidlRTCopyWinMDToOutputDirectoryDependsOn>
+    </PropertyGroup>
+
+    <!--Workaround for MidlRT nupkg expecting project to publish $(RootNamespace).winmd-->
+    <Target Name="CopyWinRTInteropWinMD" Inputs="$(CsWinRTInteropMetadata)" Outputs="$(MidlRTProjectWinMD)">
+        <Copy UseHardlinksIfPossible="$(MidlRTUseHardlinksIfPossible)"
```

Comment:
Looks like this is causing a build break?

Reply:
yes, path was wrong